packages/gestalt/src/HelpButton.js

```diff
@@ -0,0 +1,200 @@
+
+  const toggleView = () => {
+    setIsOpen((currVal) => !currVal);
+    setActive((currVal) => !currVal);
+  };
+
+  const onHandleTap = (props) => {
+    toggleView();
+    if (onClick) {
+      onClick(props);
+    }
+  };
+
+  const bgIconColor = isHovered || isActive || isFocused ? 'default' : 'subtle';
+
+  const tooltipZIndex = zIndex ?? new FixedZIndex(1);
+
+  const zIndexLayer = new CompositeZIndex([new FixedZIndex(tooltipZIndex.index())]);
+
+  const textElement = typeof text === 'string' ? <Text align="center">{text}</Text> : text;
+
+  return (
+    <Flex alignItems="center" justifyContent="center" flex="none">
+      <Tooltip
+        text={accessibilityTooltipMessage}
+        idealDirection={idealDirection}
+        zIndex={tooltipZIndex}
+      >
+        <TapArea
+          accessibilityExpanded={isOpen}
+          accessibilityControls={popoverId}
+          accessibilityLabel={accessibilityTooltipMessage}
+          fullWidth={false}
+          onTap={onHandleTap}
+          ref={ref}
+          role="button"
+          rounding="circle"
+          onBlur={() => {
+            setFocused(false);
+            setActive(false);
+          }}
+          onFocus={() => setFocused(true)}
+          onMouseDown={() => setActive(true)}
+          onMouseEnter={() => setHovered(true)}
+          onMouseLeave={() => setHovered(false)}
+          onMouseUp={() => setActive(false)}
+          tabIndex={0}
+        >
+          <Box
+            width={16}
+            height={16}
+            rounding="circle"
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+            dangerouslySetInlineStyle={{
+              __style: {
+                backgroundColor: `var(--color-text-${bgIconColor})`,
+              },
+            }}
+          >
+            <Icon
+              accessibilityLabel={accessibilityIcon}
+              color="inverse"
+              icon="question-mark"
+              size={8}
+            />
+          </Box>
+        </TapArea>
+      </Tooltip>
+      {isOpen && (
+        <Box zIndex={zIndexLayer}>
+          <Popover
+            id={popoverId}
+            accessibilityLabel={accessibilityPopoverLabel}
+            anchor={ref.current}
+            onDismiss={() => {
+              toggleView();
+              setActive(false);
+            }}
+            idealDirection={idealDirection ?? 'down'}
+            positionRelativeToAnchor
+          >
+            <Box
+              padding={4}
+              rounding={4}
+              minWidth={230}
+              onBlur={() => {
+                setFocused(false);
+                if (!link?.href) {
+                  setIsOpen(false);
+                }
+              }}
+              onFocus={() => setFocused(true)}
+            >
+              <Box tabIndex={0}>{textElement}</Box>
+              {link?.href && (
+                <Box padding={3} width="100%" display="block" marginTop={3}>
```

Comment:
marginTop={3} could potentially be replaced with Flex gap={3} direction column here

Reply:
Not works as we chat about.